src/renault_api/kamereon/models.py

```diff
@@ -49,13 +49,17 @@
     "X101VE": {  # ZOE phase 1
         "reports-in-watts": True,
         "support-endpoint-location": False,
+        "support-endpoint-lock-status": False,
```

Comment:
Can you set this on GATEWAY_SPECIFICATIONS also?

Reply:
Done